pkgs/development/libraries/libzen/default.nix

```diff
@@ -1,12 +1,12 @@
 { stdenv, fetchurl, automake, autoconf, libtool, pkgconfig }:
 
-let version = "0.4.32"; in
+let version = "0.4.33"; in
 
 stdenv.mkDerivation {
   name = "libzen-${version}";
   src = fetchurl {
-    url = "http://mediaarea.net/download/source/libzen/${version}/libzen_${version}.tar.bz2";
-    sha256 = "0rhbiaywij6jj8d7vkc4v7y21ic1kv9fbn9lk82mm12yjwzlhhyd";
+    url = "https://mediaarea.net/download/source/libzen/${version}/libzen_${version}.tar.bz2";
+    sha256 = "0py5iagajz6m5zh26svkjyy85k1dmyhi6cdbmc3cb56a4ix1k2d2";
   };
 
   buildInputs = [ automake autoconf libtool pkgconfig ];
```

Comment:
Have a look at your other PR `{lib}mediainfo{-gui}` for native vs normal `buildInputs` (I think these are all `nativeBuildInputs`).